Facebook.Android/component/component.yaml

```diff
@@ -15,8 +15,8 @@ libraries:
     - ../output/Xamarin.Facebook.AudienceNetwork.dll
 packages:
   android:
```

Comment:
Yeah this is being unnecessarily explicit.  The other dependencies will come in automatically.  Will simplify this bit.